content/sql/concepts/commands/terms/top/top.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'TOP'
+Description: 'Returns a specified number of rows from the top of the result.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'SQL Server'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Returns a specified number of rows from the top of the result.
+
+## Syntax
+
+This command is used to select the inital rows from a table, limiting the result to a specified number, represented here by (n):
+
+```sql
+SELECT TOP (n) column_name(s)
+FROM table_name;
+```
+
+The command can also be used with PERCENT to limit the result to the top (n) percent of rows:
```

Comment:
```suggestion
The command can also be used with PERCENT to limit the result to the top `n` percent of rows:
```

Reply:
To add to this suggestion, let's surround "PERCENT" in backticks (i.e., \`PERCENT\`)